build/includes/sdk.mk

```
@@ -22,12 +22,14 @@
 
 build_sdk_base_version = $(call sha,$(build_path)/build-sdk-images/tool/base/Dockerfile)
 build_sdk_base_tag = agones-build-sdk-base:$(build_sdk_base_version)
+build_sdk_version = $(call sha_bash,$(build_path)/build-sdk-images/$(SDK_FOLDER)/*.sh)$(call sha,$(build_path)/build-sdk-images/$(SDK_FOLDER)/Dockerfile)
```

```suggestion
build_sdk_version = $(call sha_bash,$(build_path)/build-sdk-images/$(SDK_FOLDER)/{*.sh,Dockerfile})
```

`sha256sum {*.sh,Dockerfile} | cut -d' ' -f1 | sha256sum | head -c 10` - works in bash!

---

Thanks, let's use just `$(SDK_FOLDER)/*` because suggested version leads to :  
```
sha256sum: '"/home/UserName/go/src/agones.dev/agones/build//build-sdk-images/go/{*.sh': No such file or directory
```

---

I mean we could even more simplify to:
```
sha256sum {*} | cut -d' ' -f1 | sha256sum | head -c 10
```

---

Yes - we could use everything in the directory. We may end up with a README.md in each one, but that's probably fine if we happen to catch that. WDYT?

---

Yes, the proper wildcard should fix README.md case in the future.

---

I think it's fine if we end up pickup up a README - also it's not an issue _right now_ so, let's just use a solution that simplest for now (use `build_sdk_version = $(call sha_bash,$(build_path)/build-sdk-images/$(SDK_FOLDER)/*`) and we can always revisit it when we actually need to.

WDYT?

---

Yes, that would work, for now we don't have README files in any of `build-sdk-images/$(SDK_FOLDER)/`, I will update PR with suggested naming.